dependabot_file.py

```diff
@@ -4,23 +4,33 @@
 import yaml
 
 
-def make_dependabot_config(ecosystem, group_dependencies, indent) -> str:
+def make_dependabot_config(
+    ecosystem, group_dependencies, indent, schedule, schedule_day
+) -> str:
     """
     Make the dependabot configuration for a specific package ecosystem
 
     Args:
         ecosystem: the package ecosystem to make the dependabot configuration for
         group_dependencies: whether to group dependencies in the dependabot.yml file
         indent: the number of spaces to indent the dependabot configuration ex: "  "
+        schedule: the schedule to run dependabot ex: "daily"
+        schedule_day: the day of the week to run dependabot ex: "monday" if schedule is "daily"
```

Comment:
If the schedule is daily, we wouldn't set the day of the week, right? Suggested a change on this line to weekly.
```suggestion
        schedule_day: the day of the week to run dependabot ex: "monday" if schedule is "weekly"
```